features/facebookImport/src/views/overview/overview.jsx

```diff
@@ -52,6 +54,16 @@ const Overview = () => {
         return b.value - a.value;
     });
 
+    const formatTime = (time) => {
```

Comment:
What we had under _util/formatTime_ was a bit overengineered :D It supported all kinds of use cases we never actually use. This is 100% functionally equivalent to what we have, believe it or not. Without options nobody uses, that is.